test-common/src/main/java/com/synopsys/integration/alert/test/common/TestResourceUtils.java

```diff
@@ -21,9 +21,23 @@
 
 public final class TestResourceUtils {
     public static final String DEFAULT_PROPERTIES_FILE_NAME = "test.properties";
+    public static final String DEFAULT_SWAGGER_API_SPEC_FILE_NAME = "swagger.api-spec";
 
     private static final String SUB_PROJECT_NAME = "test-common";
-    private static final File EXPECTED_BASE_TEST_RESOURCE_DIR = new File(TestResourceUtils.class.getProtectionDomain().getCodeSource().getLocation().getPath(), "../../../../src/main/resources/");
+    private static final File EXPECTED_BASE_TEST_RESOURCE_DIR = new File(
+        TestResourceUtils.class.getProtectionDomain().getCodeSource().getLocation().getPath(),
+        "../../../../"
+    );
```

Comment:
Changing of this path to remove `/src/main/resources` is done because this is a bug. `EXPECTED_BASE_TEST_RESOURCE_DIR` is used if the `buildOutputDir` is not found. But either way, found or not, we append `/src/main/resources` to the end of the path. So if `buildOutputDir` isn't found, the returned value would have been `../../../../src/main/resources/src/main/resources`